### ISSUE TITLE
Security alerts 206 w/ no content

### ISSUE THREAD

**Issue by jt-aldridge** [microsoftgraph/microsoft-graph-explorer-v4#1266]
The graph explorer (`https://developer.microsoft.com/en-us/graph/graph-explorer`) used to return data when hitting the  `https://graph.office.net/en-us/graph/api/proxy?url=graph.microsoft.com/v1.0/security/alerts` endpoint with the auth token `https://graph.microsoft.com/`. This no longer happens. Instead a 206 is returned. The data being returned was predictable and so could be used for testing. This behaviour is also exhibited on the graph api webpage, i.e. It used to return a selection of security alerts but now returns a 206.

**Describe the bug**
The GET https://graph.microsoft.com/v1.0/$metadata#security/alerts of the graph explorer no longer returns any events. This is used for checking pipelines that process alerts. Previously the endpoint returned a set of events when interacted with.

**To Reproduce**
GET https://graph.microsoft.com/v1.0/$metadata#security/alerts on the graph explorer webpage

**Expected behavior**
The GET https://graph.microsoft.com/v1.0/$metadata#security/alerts on the graph explorer webpage should return security sample security alerts

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Fedora35
 - Browser Firefox
 - Version 94

**Smartphone (please complete the following information):**
N/A

**Additional context**
This beahviour was changed as of 16/11/2021

**Comment by millicentachieng** [comment]
@jt-aldridge I want to think that the query you are referring to is [https://graph.microsoft.com/v1.0/security/alerts](https://developer.microsoft.com/en-us/graph/graph-explorer?request=security/alerts&method=GET&version=v1.0&GraphUrl=https://graph.microsoft.com).

Here's a [description](https://developer.microsoft.com/en-us/graph/graph-explorer?request=security/alerts&method=GET&version=v1.0&GraphUrl=https://graph.microsoft.com) of the 206 status code you received. There seems to be an error in a response from one of the data providers. The response you were expecting is in the `warning` header which is not displayed on Graph Explorer since it's not part of the [exposed headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers).

**Comment by jt-aldridge** [comment]
@millicentachieng Sorry I made a complete misstep in explaining the bug. I have edited the initial response and will outline in better detail here. The bug is to do with the Graph API explorer.

The graph explorer (`https://developer.microsoft.com/en-us/graph/graph-explorer`) used to return data when hitting the  `https://graph.office.net/en-us/graph/api/proxy?url=graph.microsoft.com/v1.0/security/alerts` endpoint with the auth token `https://graph.microsoft.com/`. This no longer happens. Instead a 206 is returned. The data being returned was predictable and so could be used for testing. This behaviour is also exhibited on the graph api webpage, i.e. It used to return a selection of security alerts but now returns a 206.
